src/basic/path-util.c

```diff
@@ -657,7 +657,14 @@ static int find_executable_impl(const char *name, const char *root, char **ret_f
         return 0;
 }
 
-int find_executable_full(const char *name, const char *root, char **exec_search_path, bool use_path_envvar, char **ret_filename, int *ret_fd) {
+int find_executable_full(
```

Comment:
Not a matter, but I'd like to split out such not mentioned change into another commit.

Reply:
I split them out to a separate commit. I didn't do that before because these changes I wanted to avoid a trivial commit.